### PR TITLE
added variable cli input and added models folder

### PR DESCRIPTION
**Pull request by mz106**
Added yargs functionality

### CODE REVIEW

File: models/cat.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const mongoose = require("mongoose");
+
+
```
Comment (nmunro) [inline models/cat.js:3]: Un-needed extra white space here

File: index.js [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
 (async () => {
     mongoose.connect(process.env.MONGO_URI);
 
-    const Cat = mongoose.model("Cat", {
-        name: { type: String, unique: true }
-    });
+    // const Cat = mongoose.model("Cat", {
+    //     name: { type: String, unique: true },
+    //     age: { type: Number, unique: false }
+    // });
```
Comment (nmunro) [inline index.js:16]: This exists in your model file now, can we remove this?

**Review by nmunro — CHANGES_REQUESTED** [review]
Pretty good, but can we address my comments, please? 👍

File: index.js (hunk 2 above)
Comment (mz106) [reply index.js]: Removed model from index.js

**Review by nmunro — APPROVED** [review]
Thanks, appreciate it!

**Pull request merged**